### PR TITLE
update airspace files to more recent versions

### LINKED ISSUE #381: Update Dutch airspace for 2015
New airspace can be downloaded, in OpenAir format, at this URL:
https://www.gliderpilotshop.com/databases

### LINKED ISSUE #435: airspace data needs update
I just noticed that airspace data for Germany is outdated. For instance EDR138, EDR144, EDR147.
How can this be updated to avoid false alarms?

### LINKED ISSUE #639: outdated airspace
to my knowledge, these forbidden areas near Borkenberge are no longer there:
screen shot 2017-04-29 at 23 51 54

also seems like the dutch airspace is outdated, the Niederein corridor extends into the Netherlands in current airspace.
newest dutch airspace file can be found here: [cub](https://www.dropbox.com/s/xvb1044zgfz0afs/EHv17_3.cub?dl=0)/[txt](https://www.dropbox.com/s/713g4uehyu8il9o/EHv17_3.txt?dl=0).
screen shot 2017-04-29 at 23 53 35

### PR DESCRIPTION
**Pull request by GliderGeek**
This pull request updates the airspace files to the most recent versions found on soaringweb.org.

This changeset encompasses:
Germany, Belgium, France, Netherlands, Switzerland, Spain, Poland, Norway, Denmark, Finland

most are updated to 2017

This pull request closes #639, closes #381, closes #435

### CODE REVIEW

File: mapserver/airspace/airspace_list.txt [hunk 1]
```diff
@@ -6,22 +6,22 @@
...
-de openair http://www.daec.de/fileadmin/user_upload/files/2012/fachbereiche/luftraum/20140306OpenAir.txt
+de openair http://soaringweb.org/Airspace/DE/Germany_CW13_2017.txt
 at openair http://www.austrocontrol.at/jart/prj3/austro_control/data/uploads/austria_ATS_2011_openair.txt
 # current belgium airspace can be downloaded from http://www.lvzc.be/index.php/secretariaat/documenten/cat_view/53-luchtruim/54-luchtruimbestanden as zip file
-be openair http://www.zweefvliegen.be/index.php/secretariaat/documenten/doc_view/322-bellux-week-120112
-fr openair http://soaringweb.org/Airspace/FR/140708__AIRSPACE_France_1405e.txt
-nl openair http://www.gliderpilotshop.nl/documents/EHv14_5.txt
+be openair http://soaringweb.org/Airspace/BE/BELLUX_WEEK_170325b.txt
+fr openair http://soaringweb.org/Airspace/FR/170529__AIRSPACE_FRANCE_1705.txt
+nl http://soaringweb.org/Airspace/NL/EHv17_3.txt
```
Comment (Turbo87) [inline mapserver/airspace/airspace_list.txt]: seems like `openair` got removed here by mistake?

Comment (GliderGeek) [inline mapserver/airspace/airspace_list.txt]: yes that is correct, will fix that in separate commit

**Comment by GliderGeek** [comment]
Can someone help out on checking whether these new airspace files are handled correctly?
I currently dont have a local installation work on which this can be tested.

**Comment by Turbo87** [comment]
> Can someone help out on checking whether these new airspace files are handled correctly?
> I currently dont have a local installation on which this can be tested.

unfortunately I don't have the airspace rendering code setup locally either which is why I haven't merged this yet... 😞

**Comment by niggli** [comment]
French link is already broken again, current one is http://soaringweb.org/Airspace/FR/170922__AIRSPACE_France_2017-08-17.txt

**Comment by Turbo87** [comment]
Finally managed to get this done! 🎉 

I've updated some more airspaces and updated the PR :)

**Comment by GliderGeek** [comment]
Awesome stuff @Turbo87! Did you get the mapserver running locally?

**Comment by Turbo87** [comment]
Nope, but I broke prod for a few hours :see_no_evil:

**Comment by GliderGeek** [comment]
ouch. any idea how we could get this thing working / ask someone who can guide us in the right direction?

**Comment by Turbo87** [comment]
The thing that broke was actually not related to the airspace (except for Belgium), it was some MapServer update that broke the airfield rendering (see https://github.com/skylines-project/skylines/pull/770)

**Pull request merged**